sphinx_gallery/downloads.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+r"""
+Utilities for downloadable items
+================================
+
+"""
+# Author: Óscar Nájera
+# License: 3-clause BSD
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import zipfile
+
+CODE_DOWNLOAD = """**Total running time of the script:**
+({0:.0f} minutes {1:.3f} seconds)\n\n
```

Comment:
I find it quite surprising to have the running time of the script part of the download template.